explainer.md

```diff
@@ -1,7 +1,63 @@
 # WebXR Device API Explained
+The [WebXR Device API](https://immersive-web.github.io/webxr/) provides access to input and output capabilities commonly associated with Virtual Reality (VR) and Augmented Reality (AR) devices. It allows you develop and host VR and AR experiences on the web.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
```

Comment:
should this state the exact command "npm run doctoc"?  Or is it also autogenerated?

Reply:
Autogenerated.